speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/DecisionMaker.inl

```diff
@@ -25,9 +26,7 @@ template <typename SocketBlackBox, typename SDAConfig>
 void DecisionMaker<SocketBlackBox, SDAConfig>::Initialize(tCarElt* p_initialCar, 
     tSituation* p_initialSituation, BlackBoxData* p_testSituations, int p_testAmount)
 {
-#ifdef BB_RECORD_SESSION
     m_recorder = new Recorder("BB_Recordings", "bbRecording", 2);
```

Comment:
the making of the recorder should be in the if statement as well, since it will create and open a file